votor-messages/Cargo.toml

```diff
@@ -22,9 +22,12 @@ frozen-abi = [
 [dependencies]
 agave-feature-set = { workspace = true }
 agave-logger = { workspace = true }
+bitvec = { workspace = true }
+bytemuck = { workspace = true }
 log = { workspace = true }
+num_enum = { workspace = true }
 serde = { workspace = true }
-solana-address = { workspace = true, features = ["curve25519"] }
```

Comment:
i should've backported this but got lazy, we should continue using `solana-address` for things that are addresses (the GENESIS_CERTIFICATE_OFF_CURVE_ACCOUNT)